packages/libs/client/src/integration-tests/helpers/account/keyFromAccount.ts

```diff
@@ -0,0 +1,4 @@
+export type Account = `${'k' | 'w'}:${string}` | string;
+export function keyFromAccount(account: Account): string {
+  return account.split(':')[1];
+}
```

Comment:
Taking the key from the account can only be done with `k:...` accounts. Would be good to do a runtime check whether the `account` is compatible. Even when it is, the key could've been rotated. 

```suggestion
export function keyFromAccount(account: Account): string {
  if (!account.includes('k:')) {
    throw new Error(`Not able to retrieve key from '${account}'`);
  }
  return account.split(':')[1];
}
```

Reply:
@alber70g given that we provide a type for Account: `export type Account = `${'k' | 'w'}:${string}` | string;`
Perhaps we could just return the account if there is no `k:` prefix.